app/src/main/java/com/mashup/lastgarden/ui/editor/EditorFragment.kt

```diff
@@ -24,4 +37,39 @@ class EditorFragment : Fragment() {
         )
         return binding.root
     }
+
+    override fun onSetupViews(view: View) {
+        super.onSetupViews(view)
+        initEditorView()
+        observeViewModel()
+
+        binding.closeBtn.setOnClickListener {
+            requireActivity().finish()
+        }
+        binding.floatingButton.setOnClickListener {
+            findNavController().navigate(
+                R.id.actionEditorFragmentToEditorUploadFragment
+            )
+        }
+    }
+
+    private fun initEditorView() {
+        editor = PhotoEditor.Builder(requireContext(), binding.photoEditorView)
+            .setPinchTextScalable(true)
+            .setClipSourceImage(true)
+            .build()
+    }
+
+    private fun observeViewModel() {
+        editorViewModel.imageUrl.observe(this) { imageUrl ->
+            setImageUsingFileUrl(imageUrl)
+        }
+    }
+
+    private fun setImageUsingFileUrl(url: String) {
+        binding.photoEditorView.source.loadImage(
+            glideRequests,
+            url
```

Comment:
약간 약속적인 부분이긴한데 이렇게 줄이 나뉘어진 경우 앞에 parameter name을 적어주는게 좋아요.
```suggestion
            gliRequest = glideRequests,
            imageUrl = url
```

Reply:
오 좋네요